apps/web/prisma/client/schema.prisma

```diff
@@ -5,14 +5,71 @@ generator client {
   provider = "prisma-client-js"
   output = "./client"
 }
-
+// TODO: Change ID to UUID
 datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
 
 model Subscribers {
-  id          BigInt @id @default(autoincrement())
-  created_at  DateTime @default(now())
-  email       String @unique
+  id                String            @id      @default(uuid())
+  Email             String            @unique
+  Name              String
+  LinkedInUrl       String
+  GitHub            String?
+  YearsOfExperience Int
```

Comment:
Thinking about this one now...

I think it would be best to store the year that the user started working professionaly with IT.

This way we always keep this info updated by calculating currentYear - YearStartedWorking

Reply:
It makes sense. Maybe we could change to StartEmploymentYear or something like that?

Reply:
I'm gonna mark it as resolved and we discuss it later